### DISCUSSION TITLE
How to collaborate on a formulae (cbonsai 1.31.1) Homebrew/homebrew-core#130819

### DISCUSSION THREAD

**Discussion by osalbahr** [Homebrew/discussions#4605] [Tap maintenance and Homebrew development] [answered]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

              I tried to create a new PR but I am not sure how to do these steps:

- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

Should I `brew tap mhzawadi/homebrew-core` first?

The instructions in [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md) say to `brew create $URL` but not sure how to do it when a repo already exists. And [Formulae related pull request](https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request#formulae-related-pull-request) says to `git remote add <YOUR_USERNAME> https://github.com/<YOUR_USERNAME>/homebrew-core.git` but not sure how it works when the repo is not under my username.

_Originally posted by @osalbahr in https://github.com/Homebrew/homebrew-core/issues/130819#issuecomment-1608183169_

Creating this discussion to avoid cluttering the closed PR. Any help would be appreciated!

**Comment by SMillerDev** [comment]
You can't commit to other people's pull request so you'll have to copy the content and then follow the brew documentation

**Reply by osalbahr** [reply]
I have push access to their fork, if that’s what you’re asking.

Are you suggesting I should use my own fork?

**Reply by SMillerDev** [reply]
Yes, the pull request is closed so there is no benefit to pushing there that I can see

**Reply by osalbahr** [reply]
I agree. To clarify, I’m trying to figure out if I can make a new PR from the same branch. The problem is, I don’t know how to have the changes be reflected locally so I can perform the required tests.

**Reply by osalbahr** [reply]
Actually, it looks like that branch might be deleted so I’d have to create a new branch anyways.

**Comment by osalbahr** [comment, marked as answer]
I decided to just copy the changes to a new branch under my fork and use git’s Co-Author feature (Homebrew/homebrew-core#135198).
